.github/workflows/gcp-experiment-pipeline.yml

```diff
@@ -0,0 +1,193 @@
+---
+name: GCP-Experiment-Pipeline
+on:
+  workflow_dispatch:
+    inputs:
+      goExperimentImage:
+        default: "litmuschaos/go-runner:ci"
+      operatorImage:
+        default: "litmuschaos/chaos-operator:ci"
+      runnerImage:
+        default: "litmuschaos/chaos-runner:ci"
+      chaosNamespace:
+        default: "default"
+      experimentImagePullPolicy:
+        default: "Always"
+
+jobs:
+  GCP_VM_Instance_Stop:
+    runs-on: ubuntu-latest
+    steps:
+
+      #Install and configure a k3s cluster
+      - name: Installing Prerequisites (K3S Cluster)
+        env: 
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+        run: |
+          curl -sfL https://get.k3s.io | sh -s - --docker --write-kubeconfig-mode 664
+          kubectl wait node --all --for condition=ready --timeout=90s
+          mkdir -p $HOME/.kube && cat /etc/rancher/k3s/k3s.yaml > $HOME/.kube/config          
+          kubectl get nodes
+          
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+
+      - name: Create Kubernetes secret for gcp experiment
+        if: always()
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+        run: |
+          cat <<EOF | kubectl apply -f -
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: cloud-secret
+          type: Opaque
+          stringData:
+            type: "service_account"
+            project_id: "${{ secrets.GCP_PROJECT_ID }}"
+            private_key_id: "${{ secrets.PRIVATE_KEY_ID }}"
+            private_key: ${{ secrets.PRIVATE_KEY }}
+            client_email: "${{ secrets.CLIENT_EMAIL }}"
+            client_id: "${{ secrets.CLIENT_ID }}"
+            auth_uri: "${{ secrets.AUTH_URI }}"
+            token_uri: "${{ secrets.TOKEN_URI }}"
+            auth_provider_x509_cert_url: "${{ secrets.AUTH_PROVIDER_CERT_URL }}"
+            client_x509_cert_url: "${{ secrets.CLIENT_CERT_URL }}"
+          EOF
+
+      - name: Set up Google Cloud SDK
+        if: always()
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - name: Create target GCP VM Instances
+        if: always()
+        run: |
+          gcloud compute instances create ${{ secrets.INSTANCE_ONE }} ${{ secrets.INSTANCE_TWO }} \
```

Comment:
we can derive the instance name here only than passing it from secrets like
```bash
gcloud compute instances create litmus-e2e-first-${{ github.run_number }}  litmus-e2e-second-${{ github.run_number }} \
```

Same for the disk name.

Reply:
Updated